src/index.html

```diff
@@ -120,6 +120,7 @@ <h1>AngularJS Multiselect</h1>
                     <div class="row">
                         <div class="col-md-6">
                             <amo-multiselect
+                                conjunction-text="or"
```

Comment:
This example looks like:

<img width="344" alt="Screen Shot 2020-07-08 at 11 59 23 AM" src="https://user-images.githubusercontent.com/847532/86942942-cf056c80-c113-11ea-9da1-2829b35d8362.png">